src/controllers/user/index.js

```diff
@@ -0,0 +1,87 @@
+import { CognitoUser, AuthenticationDetails } from 'amazon-cognito-identity-js'
+import UserPool from './user-pool'
+global.fetch = require('node-fetch')
+
+const authenticate = (Username, Password) =>
```

Comment:
When you move this to a separate file, this would be simpler:
```javascript
const authenticate = (username, password) => cognitoService.authenticate(username, password)
```

Reply:
I  moved the cognito authenticate and used the called method as a constant inside the login of the user controller